build.yaml

```diff
@@ -243,25 +251,26 @@ common_cli:
     # We have a devpi cache in AWS which we will consume instead of going
     # upstream to the PyPi servers.
     # We specify that devpi caching server using -i \$PIP_INDEX_URL
-    # which requires as to include --trusted_host as we are not (yet) using
+    # which requires as to include --trusted-host as we are not (yet) using
     # SSL on our caching box.
     # The --trusted-host option is only available with pip 7.
     #
+    if [ \${PIP_INDEX_URL} ]; then
+        PIP_ADDITIONAL_OPTIONS="\${PIP_ADDITIONAL_OPTIONS} -i \${PIP_INDEX_URL} "
```

Comment:
Is it possible to just upgrade pip here and then use the `--trusted-host` option knowing that it will be available?